### ISSUE TITLE
PHP 8.1 - Throwing deprecated warnings

### ISSUE THREAD

**Issue by benjaminprojas** [php-stubs/woocommerce-stubs#16]
I'm seeing these errors when running PHPStan with WooCommerce Stubs:

```
PHP Deprecated:  Optional parameter $id declared before required parameter $object is implicitly treated as a required parameter in .../vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php on line 73141
```

**Comment by szepeviktor** [comment]
Hello @benjaminprojas!
Yes, these problems may occur.
_made by WooCommerce_

What is your proposal?

**Comment by benjaminprojas** [comment]
This can be fixed by declaring `$object` as "optional" by setting its default value to an empty array. See #18.

**Comment by szepeviktor** [comment]
Here is the solution. In this commit

https://github.com/woocommerce/woocommerce/pull/34396/commits/5a484b76c3dd396f1b8541004aa7058cfa3d1467

https://github.com/woocommerce/woocommerce/pull/34396

Hold on tight!
WC 7.1.0 will be released soon 🎉

**Comment by benjaminprojas** [comment]
Oh boy 🤦🏻 I feel bad for anyone using that public method and updates without understanding the consequences!

Will look for this fix then once WC 7.1.0 is out 😄
